pkg/web/cookie/cookie.go

```diff
@@ -97,23 +111,9 @@ func (d *Cookie) Remove(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	http.SetCookie(w, &http.Cookie{
-		Name:     d.Name,
-		Path:     d.Path,
-		HttpOnly: d.HTTPOnly,
-		Value:    tombstone,
-		Expires:  time.Unix(1, 0),
-		MaxAge:   0,
-	})
-
-	// Additionally, explicitly remove the cookie also from the current path.
-	// This can be necessary to also remove old cookies when the cookie paths
-	// have been changed.
-	http.SetCookie(w, &http.Cookie{
-		Name:     d.Name,
-		HttpOnly: d.HTTPOnly,
-		Value:    tombstone,
-		Expires:  time.Unix(1, 0),
-		MaxAge:   0,
-	})
```

Comment:
This is no longer needed. We haven't issued such cookies for a really long time.